README.md

```diff
@@ -82,38 +82,20 @@ Most packages in the workspace have their own README file that provides specific
 
 NOTE: Some packages may have additional commands or require different setup steps, for example in a certain package the development mode may be started with `yarn start:dev` instead of `yarn start`, or `yarn dev`. For the steps below we will be assuming defaults.
 
-### Development Mode
+### Development watch mode
 
-Development mode allows you to run a package in a local development environment. This is useful for testing changes and new features before deploying them to production. Any changes made will be reflected in real-time on save, so you can see the effects of your changes immediately. These are propogated in real time to the sample app, if it is running, for that SDK too.
-
-Development mode for sdk packages will tend to use rollup and its watch mode to rebuild the package on file changes. This build output, stored in the dist folder of the sdk package, is what is exposed to any other packages that depend on it, including the sample app.
-
-To run a package in development mode, you can use the following command in the context of that package:
+Build a package and start watch mode for a package:
 
 ```bash
-# Context: Specified package
-yarn start
+yarn w YOUR_PACKAGE_NAME
 ```
 
-Some packages also come with a sample app that you can run in development mode. To run the sample app, you first need to run the SDK it depends on in development mode, and then run the sample app development mode in a seperate terminal. For example, to run the `checkout-sdk-sample-app` located at `/packages/checkout/sdk-sample-app`, you can use the following commands:
-
+e.g. for @imtbl/passport package
 ```bash
-yarn workspace @imtbl/checkout-sdk start
-
-# Or from the root of the monorepo
-cd packages/checkout/sdk
-yarn start
+yarn w @imtbl/passport
```

Comment:
I tried this and got

```
~/dev/ts-immutable-sdk try-out-nx-watch-mode                                                                                                                                                          
❯ yarn w @imtbl/dex-sdk
Usage Error: Couldn't find a script named "w".

$ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] <scriptName> ...
```

Did I do it wrong?

Reply:
`yarn run dev @imtbl/dex-sdk` seems to be the correct command.

Reply:
When running this command, I made a change in dex-sdk to introduce a type error. The watcher rebuilt dex-sdk, but said said everything was fine. Is it supposed to fail?

Reply:
My last comment is: this is great, I can see what problem it solves, thank you. Maybe you're already planning this, but what I REALLY want is to be able to run the unit tests for a package in watch mode, with all dependencies also being rebuilt. That would be a thing of beauty.

<img src="https://media4.giphy.com/media/3o8doT9BL7dgtolp7O/giphy.gif"/>

Reply:
I'll fix the doc and see if I can reproduce your problem in dex-sdk. which file you changed btw? @keithbro-imx 

for test and linting, they are coming.

Reply:
You can change any file to see the behaviour. I think the problem is that SWC transpiles the code but doesn't typecheck it - so in a sense this is working as expected given the choice of using SWC.

Reply:
I think the IDE will catch it and the typecheck script on CI will catch it. case edge enough for us to not cater for this scenario?

Reply:
Yeah I think that's fine